src/CommunityToolkit.Maui.Core/Primitives/Defaults/RatingShape.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace CommunityToolkit.Maui.Core.Primitives.Defaults;
+
+/// <summary>
+/// The type of the shape to display on the rating view
+/// </summary>
+public readonly partial struct RatingShape(string pathData)
+{
+	/// <summary>
+	/// the data path data from the svg
+	/// </summary>
+	public string PathData { get; } = pathData;
+}
+
+public partial struct RatingShape
+{
+	/// <summary>
+	/// the default star shape
+	/// </summary>
+	public static RatingShape Star => new(PathShapes.Star);
+
+	/// <summary>
+	/// The default heart shape
+	/// </summary>
+	public static RatingShape Heart => new(PathShapes.Heart);
```

Comment:
Let's use readonly properties instead of expression bodied properties. 

A read-only property will be initialized once. Whereas an expression-bodied property returns a new instance, e.g. `new RatingShape(PathShapes.Star)`, each time it is called.
```suggestion
	public static RatingShape Heart { get; } = new(PathShapes.Heart);
```